core/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>com.yahoo.imapnio</groupId>
         <artifactId>imapnio</artifactId>
-        <version>3.0.12</version>
+        <version>3.0.14</version>
```

Comment:
13?

Reply:
check the description above, they have distributed 13 but code still show 12 not sure why.

Reply:
Your PR has conflicts. Maybe that's the reason?

Reply:
resolved